common-app-lib/src/cmr/common_app/services/ingest/subscription_common.clj

```diff
@@ -0,0 +1,15 @@
+(ns cmr.common-app.services.ingest.subscription-common
+  "This contains the code for the scheduled subscription code to be shared
+   between metadata-db and ingest."
+  (:require
+   [clojure.string :as string]))
+
+(defn normalize-parameters-v1
+  "Take a list of URL parameters, and sort them."
```

Comment:
Change the docstring to say something like "Returns the normalized parameter url string by ....".

Reply:
Updated to, "Returns a normalized url parameter string by splitting the string of parameters on '&' and sorting them alphabetically"